mslib/msui/mscolab.py

```diff
@@ -265,7 +267,8 @@ def login_handler(self):
 
         if r.text == "False":
             # show status indicating about wrong credentials
-            self.set_status("Error", 'Oh no, your credentials were incorrect.')
+            self.set_status("Error", 'Oh no, your credentials were incorrect. '
+                            f'<a href="{url_recover_password}">Recover Password</a>')
```

Comment:
here are two options possible.

The user could also have not an account.  Maybe we should change the message at all. 

'Oh no, you need to add a user account or you need to {url_recover_password}">Recover Your Password</a>'